core/evtforward/ethcall/spec_test.go

```diff
@@ -32,7 +34,13 @@ func TestSpec(t *testing.T) {
 		Every:   5,
 	}
 
-	originalSpec := ethcall.NewSpec(call, trigger)
+	normaliser := ethcall.EthDecimalsNormaliser{
+		Decimals: 5,
+	}
+
+	filter := ethcall.CallResultFilter{Filters: types.DataSourceSpecFilters{}}
+
+	originalSpec := ethcall.NewSpec(call, trigger, filter, normaliser)
```

Comment:
Do we also need to pass in a `requiredConfirmations` here to flex that field going to/from protos?